pkg/controller/persistentvolume/persistentvolume_index.go

```diff
@@ -24,34 +24,9 @@ import (
 	"k8s.io/kubernetes/pkg/client/cache"
 )
 
```

Comment:
minor point, but given that this is in `persistentvolume` package, the files don't need to be named `persistentvolume_index`.  This could probably have been just "index.go".